addons/iap_mail/static/src/scss/iap_mail.scss

```diff
@@ -0,0 +1,3 @@
+.o_partner_autocomplete_enrich_info i {
+    min-width: fit-content;
+}
```

Comment:
missing EOF ^^

Reply:
While shorter, reviewers may prefer to define a class and put it on every `<i>` instead of this solution applying on the whole template. Right now, it works though, so we can keep it this way. We'll see ^^